waterlog/src/Components/TankComponent.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchTankLevelById } from '../actions/TankLevelsByIdAction';
+import "../Stylesheets/_tank.scss";
+
+export class TankComponent extends Component {
+  componentWillMount() {
+    this.props.fetchTankLevelById(this.props.id);
+  }
+
+  getTankImage = (percent) => {
+    if (percent === 100) {
+      return <img src="/images/100_tank.png" className="image"
```

Comment:
You refer to these images a lot it will be to your own benefit to extract these image paths to a const var/ object to be reused all over. 
Just think the image path or image change? 
You will be looking around like a mad chicken to find all the references but if you have it in the global it is one change